apiary.apib

```diff
@@ -1,71 +1,109 @@
-FORMAT: 1A
+yFORMAT: 1A
 HOST: https://api.surfly.com/v2/
 
+# Definition of terms
+
+Throughout the API we refer to several terms specific to Surfly.  For clarification, 
+these are defined below:
+
+Client
+ - A company that is a customer of Surfly 
+
+Agent
+ - An employee of the client company, who is available to help when the 'Start Session'
+  button is pressed
+
+User 
+ - A generic term for either the agent, leader or follower.  Simply put,  a user is someone who 
+ interacts with the Surfly application.  
+
+Leader 
+ - A leader initiates the session, and invites followers to join them.  They are normally in control of the session, but 
+  can pass this control over to the follower.
+
+Follower
+ - Is invited to join the session by the leader.
+
+UI
+ - User Interface
+
+iframe 
+ - The implementation of Surfly.  
```

Comment:
in this case, iframe refers to HTML <iframe> element, not sure if we should define it. Maybe we could also try to avoid this word at all, since we might open sessions in a separate tab instead of iframe